pkg/network/ebpf/c/tracer-stats.h

```diff
@@ -258,32 +262,46 @@ static __always_inline void handle_tcp_stats(conn_tuple_t* t, struct sock* sk, u
     update_tcp_stats(t, stats);
 }
 
-#if defined(COMPILE_RUNTIME) || defined(COMPILE_PREBUILT)
-
-static __always_inline int sk_buff_to_tuple(struct sk_buff *skb, conn_tuple_t *t);
+static __always_inline int handle_skb_consume_udp(struct sock *sk, struct sk_buff *skb, int len) {
+    u64 pid_tgid = bpf_get_current_pid_tgid();
+    udp_recv_sock_t *st = bpf_map_lookup_elem(&udp_recv_sock, &pid_tgid);
+    if (!st) { // no entry means a peek
```

Comment:
I think the `len` check should come first, because it is cheaper than a map lookup.